Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py

```diff
@@ -702,7 +703,7 @@ def list_layer_version_command(args, aws_client):
         'CompatibleRuntime': args.get('compatible-runtime'),
         'Marker': args.get('token'),
         'MaxItems': arg_to_number(args.get('limit')),
-        'CompatibleArchitecture': args.get('compatible-architecture')
+        'CompatibleArchitecture': args.get('compatible_architecture')
```

Comment:
Please revert the change.
```suggestion
        'CompatibleArchitecture': args.get('compatible-architecture')
```